amity.py

```diff
@@ -0,0 +1,327 @@
+            self.allocate_office(new_person)
+            self.check_office_vacancy()
+            self.add_person_successfully(
+                name, new_id, person_title, wants_accomodation)
+            if not allocated:
+                self.unallocated.append(new_person)
+            return
+        else:
+            print "Please enter 'Fellow' or 'Staff' as the job title"
+
+    def allocate_office(self, new_person):
+        """"Allocates offices"""
+        if len(self.vacant_offices):
+            self.workspace = random.choice(self.vacant_offices)
+            self.workspace.occupants.append(new_person)
+            print new_person.name + " assigned the office " + \
+                self.workspace.room_name
+            allocated = True
+            self.check_office_vacancy()
+        else:
+            print "No vacant offices at the moment,add one or try again later"
+            return
+
+    def allocate_livingspace(self, new_person):
+        """Allocates Living spaces"""
+        vacant_livingspaces = []
+        for room in self.rooms:
+            if room.room_type == "Living" and room.isvacant:
+                vacant_livingspaces.append(room)
+        if len(vacant_livingspaces):
+            livespace = random.choice(vacant_livingspaces)
+            livespace.occupants.append(new_person)
+            print new_person.name + " assigned the livingspace " + \
+                livespace.room_name
+            allocated = True
+        else:
+            print "No vacant livingspaces at the moment," \
+                "add one or try again later"
+            return
+
+    def check_office_vacancy(self):
+        """Adds and removes rooms from vacancy lists \
+        according to status of the room"""
+        for office in self.offices:
+            space = Office(office)
+            if len(space.occupants) < space.max_occupancy:
+                if office not in self.vacant_offices:
+                    self.vacant_offices.append(office)
+                    self.vacant_rooms.append(office)
+            elif len(office.occupants) >= office.max_occupancy:
+                if office in self.vacant_offices:
+                    self.vacant_offices.remove(office)
+                    self.vacant_rooms.remove(office)
+
+    def reallocate_person(self, person_id, new_room_name):
+        """Reallocates people to new rooms"""
+        moving_person = None
+        person_id = int(person_id)
+
+        for peep in self.people:
+            if peep.person_id == person_id:
+                moving_person = peep
+
+        if moving_person is None:
+            print "That person ID does not exist, try another"
+            return
+
+        for room in self.rooms:
+            if room.room_name == new_room_name:
+                new_room = room
+
+        if new_room_name not in [room.room_name for room in self.vacant_rooms]:
+            print "The room enterred doesn't exist or is full, enter another"
+            return
```

Comment:
return the statement you are printing and then when calling this function in docopt you call it using print (func_name)